scripts/obtain/dotnet-install.sh

```diff
@@ -82,6 +82,9 @@ get_current_os_name() {
         elif [ "$(cat /etc/*-release | grep -cim1 debian)" -eq 1 ]; then
             echo "debian"
             return 0
+         elif [ "$(cat /etc/*-release | grep -cim1 gentoo)" -eq 1 ]; then
```

Comment:
This is not going to work. This function is used when calculating the URL from where to download the CLI. This will cause the script to look for a URL with gentoo somewhere in it and this will give you a 404.